blog/admin.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from django.contrib.auth.models import Group
+
+from blog.models import Post, Commentary, User
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    search_fields = ("title",)
+    list_filter = ("owner",)
+
+
+@admin.register(Commentary)
+class CommentaryAdmin(admin.ModelAdmin):
+    search_fields = ("content",)
+    list_filter = ("user",)
+
+
+admin.site.register(User, UserAdmin)
+admin.site.unregister(Group)
```

Comment:
It's generally a good idea to be consistent with how you register models. choose one approach